lib/Chat/Notifier.php

```diff
@@ -385,7 +390,7 @@ protected function shouldParticipantBeNotified(Participant $participant, ICommen
 
 		if ($participant->getSession() instanceof Session) {
 			// User is online
-			return false;
+			return $participant->getSession()->getLastPing() > $this->timeFactory->getTime() - Session::SESSION_TIMEOUT;
```

Comment:
I think is a good idea implement test to cover this scenario

Reply:
Added